src/compiler/utilities.ts

```diff
@@ -7171,30 +7170,70 @@ namespace ts {
         return d1.relatedInformation ? Comparison.LessThan : Comparison.GreaterThan;
     }
 
+    // function compareMessageText(t1: string | DiagnosticMessageChain[], t2: string | DiagnosticMessageChain[]): Comparison {

```

Comment:
Probably shouldn't leave this old code lying around here.

Reply:
woooooooops. removed.